gix-blame/src/file/function.rs

```diff
@@ -85,19 +86,34 @@ where
             .tokenize()
             .map(|token| interner.intern(token))
             .count()
-    };
+    } as u32;
 
     // Binary or otherwise empty?
     if num_lines_in_blamed == 0 {
         return Ok(Outcome::default());
     }
 
-    let mut hunks_to_blame = vec![{
-        let range_in_blamed_file = 0..num_lines_in_blamed as u32;
-        UnblamedHunk {
-            range_in_blamed_file: range_in_blamed_file.clone(),
-            suspects: [(suspect, range_in_blamed_file)].into(),
+    let range_in_blamed_file = {
+        // This block assumes that `range` has 1-based line numbers and converts it the the format
```

Comment:
I could imagine a function here like `one_based_inclusive_to_zero_based_exclusive_range()` to 'codify' the comment.